docs/jwst/straylight/main.rst

```diff
@@ -85,3 +93,25 @@ as the magnitude of the correction is typically too small to be visible from poi
 channels use only a Lorentzian kernel with the Gaussian amplitudes set to zero as such structured components are less
 obvious at these longer wavelengths.  In Channel 4 no correction appears to be necessary,
 and the amplitudes of all model components are set equal to zero.
+
+Algorithm: Residual Shower Correction
+-------------------------------------
+The residual cosmic ray shower correction relies upon the fact that cosmic rays shower artifacts tend to
+have extremely large, blobby halos that extend across one or more IFU slices and the inter-slice pixels between
+them.  These inter-slice pixels receive little to no direct light from the sky, and can be used to construct
+a model of residual showers which are coherent in detector space.
+
+Algorithmically, this routine first uses the :ref:`REGIONS <regions_reffile>` reference file to locate and
+mask all pixels on the detector that receive direct light from the sky.  Pixels that are flagged as
+DO_NOT_USE or REFERENCE_PIXEL are similarly masked, as are some highest and lowest percentile of pixel values
+to help ensure robustness against unmasked bad pixels.
+
+The unmasked remaining pixels are then used to construct a model of the detector rates in DN/s across
+both the masked and unmasked regions using a 2D Gaussian kernel.  This model of the residual shower artifacts
+is then subtracted from the data.
+
+Since the inter-slice pixels never reach truly zero throughput from the sky (particularly at short wavelengths)
+this correction is most reliable in cases for which the observations contain only faint sources and
+have had a pixel-based dedicated background subtraction applied.  While the correct can be applied to brighter
+sources, this may result in artifacts as the subtracted model may be dominated more by spilled source flux
+than by residual cosmic ray showers.
```

Comment:
While the **correct** can be applied to brighter

 Suggestion
 While the **correction** can be applied to brighter

Reply:
Fixed!